src/layouts/Layouts.tsx

```diff
@@ -0,0 +1,16 @@
+
+
+
+import { Outlet } from 'react-router-dom';
+import Sidebar from '../components/Sidebar';
```

Comment:
Xuống dòng thừa thãi

```suggestion
import Sidebar from '../components/Sidebar';
import {FC, ReactNode} from "react";
```